crates/bevy_render/src/texture/image.rs

```diff
@@ -483,9 +483,9 @@ pub trait TextureFormatPixelInfo {
 
 impl TextureFormatPixelInfo for TextureFormat {
     fn pixel_size(&self) -> usize {
-        let info = self.describe();
-        match info.block_dimensions {
-            (1, 1) => info.block_size.into(),
+        let info = self;
+        match info.block_dimensions() {
+            (1, 1) => (info.block_size(None).unwrap_or_default() as usize).into(),
```

Comment:
Same question as above here. Additionally why can we use `unwrap_or_default` here?